runtime/compiler/runtime/MetaData.cpp

```diff
@@ -226,6 +229,52 @@ createExceptionTable(
       }
    }
 
+#if defined(J9VM_OPT_MICROJIT)
+static void
+createMJITExceptionTable(
+      TR_MethodMetaData * data,
+      MJIT_ExceptionTableEntryIterator & exceptionIterator,
+      bool fourByteOffsets,
+      TR::Compilation *comp)
+   {
+   uint8_t * cursor = (uint8_t *)data + sizeof(TR_MethodMetaData);
+
+   for (TR_ExceptionTableEntry * e = exceptionIterator.getFirst(); e; e = exceptionIterator.getNext())
+      {
+      if (fourByteOffsets)
+         {
+         *(uint32_t *)cursor = e->_instructionStartPC, cursor += 4;
+         *(uint32_t *)cursor = e->_instructionEndPC, cursor += 4;
+         *(uint32_t *)cursor = e->_instructionHandlerPC, cursor += 4;
+         *(uint32_t *)cursor = e->_catchType, cursor += 4;
```

Comment:
As far as I can tell, `createMJITExceptionTable` is identical to `createExceptionTable` with two exceptions:
1) It does not call `useSpecialCatchTypeIfNecessary` which is something used just for OSR
2) It does not consider the JITServer case which is fine because I don't envision JITServer needing to do microjit compilations.

So, if microjit and OSR are incompatible (meaning that you'll never have them enabled for the same method at the same time) then we can reuse `createExceptionTable` as is.

Reply:
There is another difference in that `createMJITExceptionTable` has a parameter called `MJIT_ExceptionTableEntryIterator & exceptionIterator`.  Maybe this and `TR_ExceptionTableEntryIterator & exceptionIterator` can be folded into one iterator?